src/native/graphql_ppx.re

```diff
@@ -72,7 +72,7 @@ let drop_prefix = (prefix, str) => {
   String.sub(str, len, rest);
 };
 
-let rewrite_query = (loc, delim, query) => {
+let rewrite_query = (loc, delim, query, maybe_schema) => {
```

Comment:
Could we use labeled args here? It's getting pretty unreadable with more than 3 args.

Reply:
`maybe_schema` can then be `~schema=?`

Reply:
👍

Reply:
Because the parameter is explicitly an option in the downstream functions having it as `~schema=None` instead made things easier to read.

If you explicitly want it as `~schema=?` I can change it. But thought it didn't really matter myself

Reply:
Wait, I might have misunderstood the initial comment. Did you want all parameters to be named?

Reply:
Can be all, can be just schema 🙂